src/components/DailyTasks.js

```diff
@@ -6,16 +6,27 @@ import dailyTasks from '../db';
 
 export default function DailyTasks() {
   const [todos, setTodos] = useState(() => {
-    const value = loadFromLocalStorage('todos');
-    return value ?? dailyTasks;
+    const todosFromLocal = loadFromLocalStorage('todos');
+    const uncheckedTodos = todosFromLocal
```

Comment:
great Alice. 
if I am right, you are asking here if the local Storage isnt empty. 
try `todosFromLocal?.map(todo => ... ` instead of ` todosFromLocal ? map here : null`.
I think this should work as well and is a little shorter.

Reply:
```
 const uncheckedTodos = todosFromLocal?.map(todo =>
      todo.checkedAt !== new Date().toLocaleDateString() ? {...todo, isChecked: false, checkedAt: ''} : todo
    );
```

meinst du so? Funktionier, danke! `todosFromLocal?.map` fragt also ab, ob `todosFromLocal` true ist, und wenn false dann gibt er null aus? Wundere mich, wieso das ohne Doppelpunkt geht? Zu der Syntax finde ich keine Doku, hast du einen Link für mich? Will ich verstehen! :)

Reply:
Great that it works. 
I try to find some link for you here ;)

Reply:
I found it: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining
;)